lib/bolt/inventory/group2.rb

```diff
@@ -31,6 +31,12 @@ def initialize(data, plugins)
         raise ValidationError.new("Group name must be a String, not #{@name.inspect}", nil) unless @name.is_a?(String)
         raise ValidationError.new("Invalid group name #{@name}", @name) unless @name =~ NAME_REGEX
 
+        # DEPRECATION : remove this before finalization
+        if data.key?('target-lookups')
+          msg = "'target-lookups' are no longer a separate key. Merge 'target-lookups' and 'targets' lists and replace 'plugin' with '_plugin'." # rubocop:disable Metrics/LineLength
```

Comment:
nit, omit period at the end of this message to handle case where we have group `@name` otherwise we get:
```
'target-lookups' are no longer a separate key. Merge 'target-lookups' and 'targets' lists and replace 'plugin' with '_plugin'. for group at ["lookup-nodes"]
```